apps/omg_watcher/priv/repo/migrations/20190408131000_add_missing_indices_to_txoutputs.exs

```diff
@@ -0,0 +1,11 @@
+defmodule OMG.Watcher.Repo.Migrations.AddMissingIndicesToTxOuputs do
+  use Ecto.Migration
+
+  def change do
+    create index(:txoutputs, [:creating_txhash, :spending_txhash])
+    create index(:txoutputs, [:creating_deposit])
+    create index(:txoutputs, [:spending_txhash])
+    create index(:txoutputs, [:spending_exit])
```

Comment:
this ':spending_exit' would be super sparse - consider partial index like [here](https://github.com/omisego/elixir-omg/blob/master/apps/omg_watcher/priv/repo/migrations/20190315095855_alter_transactions_table_add_partitial_index.exs)